packages/block-library/src/post-navigation-link/edit.js

```diff
@@ -157,6 +157,8 @@ export default function PostNavigationLinkEdit( {
 			</InspectorControls>
 			<InspectorControls group="advanced">
 				<SelectControl
+					__next40pxDefaultSize
+					__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the editor, add a Post Navigation block and see the Advanced section in the block inspector.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/979f7978-75c4-4ec0-8003-5939574b8aa2" alt="Post Navigation block inspector, before" width="269">|<img src="https://github.com/user-attachments/assets/660e3fff-f02c-48bd-b9e9-70e79a25252d" alt="Post Navigation block inspector, after" width="270">|